packages/feature-store/scripts/feast-dev-portforward.sh

```
@@ -0,0 +1,636 @@
+        REACHABLE="${GREEN}reachable${NC}"
+      else
+        REACHABLE="${RED}not reachable${NC}"
+      fi
+    else
+      REACHABLE="${YELLOW}(nc not found, skipping check)${NC}"
+    fi
+
+    echo -e "  ${BLUE}${SVC}${NC}  (namespace: ${NS})"
+    echo -e "    PID      : $PID"
+    echo -e "    Port     : localhost:${LOCAL_PORT}"
+    echo -e "    Status   : $(echo -e $REACHABLE)"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)